modules/xmpp/JingleSessionPC.js

```diff
@@ -60,40 +64,42 @@ export default class JingleSessionPC extends JingleSession {
             connection,
             mediaConstraints,
             iceConfig,
+            isP2P,
+            isInitiator,
             options) {
         super(sid, me, peerjid, connection, mediaConstraints, iceConfig);
 
-        this.lasticecandidate = false;
-        this.closed = false;
-
-        this.modifyingLocalStreams = false;
-
         /**
-         * Used to keep state about muted/unmuted video streams
-         *  so we can prevent errant source-add/source-removes
-         *  from happening
+         * Stores "delayed" ICE candidates which are added to the PC once
+         * the first sRD/sLD cycle is done.
+         * @type {Array} an array of ICE candidate lines which can be added
+         * directly to the PC
          */
-        this.modifiedSSRCs = {};
+        this.candidiates = [];
```

Comment:
is this new?  maybe name them to better reflect the documentation so it's more obvious what they are when used in the code.  `delayedIceCandidates`?